tests/tests/test-balance/test-balance-extrinsics.ts

```diff
@@ -40,9 +41,11 @@ describeDevMoonbeam("Balance extrinsics", (context) => {
         // ethereum.Executed, system.ExtrinsicSuccess
         case 3:
           expect(section === "ethereum" && method === "transact").to.be.true;
-          expect(events.length === 4);
+          expect(events.length === 11).to.be.true;
```

Comment:
This seems like a weird way to write assertions. Why not `expect(events.length).to.equal(11);`?